lib/lotus/helpers/html_helper/empty_html_node.rb

```diff
@@ -48,7 +54,12 @@ def attributes
           result = [nil]
 
           @attributes.each do |name, value|
-            result << %(#{ name }="#{ value }")
+            # only present shortform `required` on certain elements
+            if name.to_s == 'required' && HTML5_REQUIRED_CONSTRAINT.include?(@name.to_s)
+              result << %(#{ name })
```

Comment:
Just append _name_? Your code currently creates a string and then interpolates _#{name}_ into it.